modules/ROOT/nav.adoc

```diff
@@ -12,6 +12,7 @@
 * xref:runtime-fabric-logs.adoc[View and Configure Logs in Runtime Fabric]
 * xref:configure-alerting.adoc[Configure Alerting on Anypoint Runtime Fabric]
 * xref:deploy-index.adoc[Deploy Applications and API Proxies to Runtime Fabric]
+ ** xref:harden-env.adoc[Perform Environment Hardening]
```

Comment:
I think "Hardening Mule" makes more sense.

Reply:
Updated - thank you!